### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BenXHub**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+* HTML5
+* CSS3
+* Bootstrap 5.1.3
+* Font Awesome 6.3.0
+* JavaScript
+* Vue.js 3.2.45
+* Lodash 4.17.21
+
+## Usage
+
```
Comment (kulotsystems) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+The following technologies were used to create this web application:
+
+* HTML5
+* CSS3
+* Bootstrap 5.1.3
+* Font Awesome 6.3.0
+* JavaScript
+* Vue.js 3.2.45
+* Lodash 4.17.21
+
```
Comment (kulotsystems) [inline README.md]: No need for this.

```suggestion
```

**Review by kulotsystems — CHANGES_REQUESTED** [review]
Thanks @benjamineke I am requesting some changes:

File: README.md (hunk 1 above)
Comment (BenXHub) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (BenXHub) [reply README.md]: Done

**Review by kulotsystems — APPROVED** [review]
LGTM!

**Pull request merged**